docs/source/release_notes.rst

```diff
@@ -13,6 +13,10 @@ Upcoming Release
   The features in this section are not released yet, but will be part of the next release! To use the features already you have to install the main branch,
   e.g. ``pip install git+https://github.com/assume-framework/assume``
 
+**New Features:**
```

Comment:
would be good to also mention the default behavior that if not specified they are not active